test/api.js

```diff
@@ -2235,6 +2240,7 @@ describe('/v1', function() {
             var result = res.result;
             assert.equal(result.length, 1);
             assert.equal(result[0].id, client1Id.toString('hex'));
+            assert.equal(result[0].lastAccessTime, tok.createdAt.getTime());
```

Comment:
move the `lastAccessTime should equal to the latest Token createdAt time`from the test title into here: 

`assert.equal(result[0].lastAccessTime, tok.createdAt.getTime(), 'lastAccessTime should equal to the latest Token createdAt time');